src/main/kotlin/dev/stalla/builder/validating/podcast/ValidatingPodcastItunesBuilder.kt

```diff
@@ -68,11 +68,12 @@ internal class ValidatingPodcastItunesBuilder : PodcastItunesBuilder {
         return PodcastItunes(
             subtitle = subtitle,
             summary = summary,
-            image = imageBuilderValue.build() ?: return null,
+            image = imageBuilderValue.build(),
```

Comment:
Getting rid of the ReturnCount, and the validation is part of `hasEnoughDataToBuild` anyway.

Reply:
I usually suppress `ReturnCount` as well, given I prefer early returns over nullability and a gazillion checks all over